matrix-room.c

```diff
@@ -1020,10 +1020,8 @@ void matrix_room_send_message(PurpleConversation *conv, const gchar *message)
         return;
     }
 
-    if (!strncmp(message, "/me ", 4)) {
+    if (purple_message_meify(message_to_send, -1))
         type_string = "m.emote";
-        message_to_send = message + 4;
-    }
```

Comment:
btw, generally I prefer to keep braces on one-line if bodies, but certainly I'll accept that the code is not a model of consistency on this front.